test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -369,7 +369,9 @@ public void AddProductCaseSensitiveTest(SupportedLanguages lang)
             Assert.Throws<AggregateException>(() => this.SendOutputGetRequest("addproduct-params", query).Wait());
 
             // Change database collation back to case insensitive
-            this.ExecuteNonQuery($"ALTER DATABASE {this.DatabaseName} COLLATE Latin1_General_CI_AS");
+            this.ExecuteNonQuery($"ALTER DATABASE {this.DatabaseName} SET Single_User WITH ROLLBACK IMMEDIATE; ALTER DATABASE {this.DatabaseName} COLLATE Latin1_General_CI_AS; ALTER DATABASE {this.DatabaseName} SET Multi_User;");
+
+            Thread.Sleep(660000); // Wait 11 minutes to ensure table info cache is not hit since database collation was changed
```

Comment:
This increases build time by a lot (especially once more languages are added). Another option is to remove the second get request since the main objective of this test is to test the case sensitive scenario.

Reply:
Instead of this we should add a way for tests to bypass/reduce this cache. I would suggest we add an environment variable for that - something like `AZ_FUNC_TABLE_INFO_CACHE_TIMEOUT_MS` (so we can set 0 to not have caching at all)